src/Schema/Types/BlockTypes.php

```diff
@@ -188,9 +195,11 @@ protected static function register_attributes_types($block_type, $prefix) {
 
 			register_graphql_union_type($type, [
 				'typeNames' => $types,
-				'resolveType' => function ($attributes) use ($types_by_definition) {
+				'resolveType' => function ($attributes) use ($types_by_definition, $type_registry, $types) {
```

Comment:
@jasonbahl Not breaking anything for me as I register all my ACF fields with PHP, but when I use the ACF custom field editor to create a new group, I get a bunch of `Undefined variable $type_registry in /Users/[redacted]/projects/php-projects/[redacted]/app/plugins/wp-graphql-gutenberg/src/Schema/Types/BlockTypes.php on line 198` errors rendered into the GraphQL settings block

<img width="1208" alt="Screenshot 2021-08-06 at 22 36 47" src="https://user-images.githubusercontent.com/631670/128573540-3c625ac5-1b90-416e-a8af-8d0ef2833918.png">

Reply:
This is using 1.6.2 of wp-graphql, and this 0.3.9 tag.

Reply:
Ah @jasonbahl yes, we can remove the `$type_registry` from that line as it's not needed here. Was left over as I was working on some updates where I thought we might need it in the `ResolveType` callback. 

I'll follow up with another quick change

Reply:
Sorry for the report-onslaught - Another little breadcrumb:
My `gatsby-source-wordpress@4` site doesn't like the graphQL schema any more either:

```sh
success  gatsby-source-wordpress  fetch root fields - 0.329s
success  gatsby-source-wordpress  pull updates since last build - 1.328s
success Checking for changed pages - 0.001s
success source and transform nodes - 1.864s

 ERROR 

Missing onError handler for invocation 'building-schema', error was 'Error: Type with name "WpBlockAttributesObject" does not exists'. Stacktrace was 'Error: Type with name "WpBlockAttributesObject" does not exists
```

Reply:
@jonsherrard, this might be related to Gatsby Source WordPress. I believe this open issue might be related: https://github.com/wp-graphql/wp-graphql/issues/2046

Reply:
Ah yes, bit late here and bamboozled, gatsby-source-wordpress hasn't worked properly since about 1.5.6-ish if I recall.

Reply:
Oh, interesting. Would be nice to pinpoint which release specifically caused this issue. 

My thought was that the 1.6.0 release of WPGraphQL which introduced Lazy Type loading was causing the issue as Gatsby would have to be a bit more specific about it's Introspection queries and re-assignment of Types in the Gatsby GraphQL Schema. 

But if it was a 1.5.* release that caused Gatsby to no longer build, then there might be another issue that I'm unaware of.

If you're able to do some testing with 1.5.4, 1.5.5, 1.5.6, 1.5.7, 1.5.8, 1.5.9 and 1.6.2, that would be 👌🏻


I do think there was an issue with 1.5.6 that caused one issue, but that was patched in 1.5.9

I believe the 1.6.* issues are different and related to how Gatsby does introspection and builds the Schema in Gatsby. 

Since WPGraphQL is doing Lazy Type loading, I _believe_ Gatsby will need to be more specific about the Types that it asks for from WPGraphQL and being more intentional about how it maps those types to the Gatsby Schema.